src/main/java/com/akamai/edgeauth/EdgeAuth.java

```diff
@@ -86,6 +86,24 @@ public class EdgeAuth {
     /** print all parameters. */
     private boolean verbose;
 
+    /**
+     * @param tokenType Not used
```

Comment:
Not used apart from setter, is it needed?

Reply:
Similar case is for Node version https://github.com/akamai/EdgeAuth-Token-Node/blob/master/lib/edgeAuth.js , so I would not change it (yet)